.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,95 @@
+name: Bug Report
+description: File a bug report to help improve qBittorrent user experience.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        #### ADVISORY
+        "We do not support any versions older than the current release series"
+
+        "We do not support any 3rd party/forked versions e.g. `portableapps`/`Enhanced Edition`etc."
+
+        "Please post all details in **English**."
+
+  - type: checkboxes
+    attributes:
+      label: Mandatory prerequisites
+      description: To rule out invalid issues, confirm and check each one of the required checkboxes.
+      options:
+        - label: I have read the issue reporting section in the **[contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md)**, so I know how to submit a good bug report with the required information.
+          required: true
+        - label: I have verified that the issue is not fixed and is reproducible in the **[latest official qBittorrent version](https://www.qbittorrent.org/download.php).**
+          required: true
+        - label: I have checked the **[frequent/common issues list](https://github.com/qbittorrent/qBittorrent/projects/2)** and performed a **[search of the issue tracker (including closed ones)](https://github.com/qbittorrent/qBittorrent/issues)** to avoid posting a duplicate.
+          required: true
+        - label: This is not a support request or question, both of which are better suited for either the **[discussions section](https://github.com/qbittorrent/qBittorrent/discussions)**, **[forum](https://qbforums.shiki.hu/)**, or **[subreddit](https://www.reddit.com/r/qBittorrent/)**. The **[wiki](https://github.com/qbittorrent/qBittorrent/wiki)** did not contain a suitable solution either.
+          required: true
+        - label: I understand that it is the responsibility of the user to **redact any sensitive information** in logs/config files.
+          required: true
+        - label: I understand that if relevant to issue/when asked, I must provide the **qBittorrent.ini**, **qBittorrent.log** and **watched_folders.json** file(s) (if using "Watched Folders" feature). See **[Where does qBittorrent save its settings?](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings)**
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Optional prerequisites
```

Comment:
Honestly i'm not a fan of the creation of a whole section for what should be 1 checkbox (more scrolling...). Who's ever going to check more than one of the CI test builds anyway? And if you say it's for our convenience, it's totally not. We'll know which CI build they tested by the user's OS. In the end, i doubt we even care which OS they tested it on.

Reply:
Hmm, that makes sense. Maybe 'Optional prerequisites' section should be removed.

Reply:
>Honestly i'm not a fan of the creation of a whole section for what should be 1 checkbox (more scrolling...). Who's ever going to check more than one of the CI test builds anyway? And if you say it's for our convenience, it's totally not. We'll know which CI build they tested by the user's OS. In the end, i doubt we even care which OS they tested it on.

I only included it in to it's own section as we had already asked about the CI builds, since we now have segregated OS builds & to separate mandatory/optional.

ref.:
> - [ ] (optional but recommended) I have verified that the **issue is not fixed and is reproducible** in the **[latest CI build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci.yaml?query=branch%3Amaster+event%3Apush)**
____

I agree with general consensus to remove 'Optional prerequisites' section & I will include the CI builds with the "official qBittorrent version"

Example:
>I have verified that the issue is not fixed and is reproducible in the **[latest official qBittorrent version](https://www.qbittorrent.org/download.php)** or in any of the relevant **[latest CI macOS build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_macos.yaml?query=branch%3Amaster+event%3Apush)**/**[latest CI Ubuntu build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_ubuntu.yaml?query=branch%3Amaster+event%3Apush)**/**[latest CI Windows build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_windows.yaml?query=branch%3Amaster+event%3Apush).**

@thalieht Is this ok with you?

Reply:
> I have verified that the issue is not fixed and is reproducible in the **[latest official qBittorrent version](https://www.qbittorrent.org/download.php)** or in any of the relevant **[latest CI macOS build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_macos.yaml?query=branch%3Amaster+event%3Apush)**/**[latest CI Ubuntu build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_ubuntu.yaml?query=branch%3Amaster+event%3Apush)**/**[latest CI Windows build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_windows.yaml?query=branch%3Amaster+event%3Apush).**
> 

I think the last part is a bit of a mouthful. How about this?? =

> 
> I have verified that the issue is not fixed and is reproducible in the **[latest official qBittorrent version](https://www.qbittorrent.org/download.php)** or in any of the relevant latest CI builds  (**[macOS build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_macos.yaml?query=branch%3Amaster+event%3Apush)**/**[Ubuntu build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_ubuntu.yaml?query=branch%3Amaster+event%3Apush)**/**[Windows build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_windows.yaml?query=branch%3Amaster+event%3Apush)**).

Reply:
>I think the last part is a bit of a mouthful. How about this?? =

> 
> I have verified that the issue is not fixed and is reproducible in the **[latest official qBittorrent version](https://www.qbittorrent.org/download.php)** or in any of the relevant latest CI builds  (**[macOS build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_macos.yaml?query=branch%3Amaster+event%3Apush)**/**[Ubuntu build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_ubuntu.yaml?query=branch%3Amaster+event%3Apush)**/**[Windows build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_windows.yaml?query=branch%3Amaster+event%3Apush)**).

Ya, I'll go with this unless there's any objections from others.

Thanks @ArcticGems

Reply:
>& to separate mandatory/optional.

There was nothing wrong with them being mixed since the optional one was denoted as such (maybe it's just me).

>& I will include the CI builds with the "official qBittorrent version"
>
>Example:
>
>>    I have verified that the issue is not fixed and is reproducible in the latest official qBittorrent version or in any of the relevant latest CI macOS build/latest CI Ubuntu build/latest CI Windows build.

That would make testing the CI builds mandatory. Are we ok with that? I think it's too much but no strong opinion from me.
If you go with that here's a shorter version (with spaces around the slashes to make it clear they are separate links):
>I have verified that the issue is not fixed and is reproducible in the latest official qBittorrent version or in the latest CI build 
(macOS / Ubuntu / Windows).

>

Reply:
Maybe we should go back to the old one (but updated) =
> 
> * [ ]  (optional but recommended) I have verified that the issue is not fixed and is reproducible in the relevant latest CI build  (**[macOS build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_macos.yaml?query=branch%3Amaster+event%3Apush)**/**[Ubuntu build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_ubuntu.yaml?query=branch%3Amaster+event%3Apush)**/**[Windows build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_windows.yaml?query=branch%3Amaster+event%3Apush)**).

I chose to keep "build" (thalieht chosed to remove it) because I feel it's more clear.

Reply:
>There was nothing wrong with them being mixed since the optional one was denoted as such (maybe it's just me).

I don't have any strong opinion on it either

>Maybe we should go back to the old one (but updated) =

What about this? (marked as not needing validation, of course!)

> * [ ]  (optional but recommended) I have verified that the issue is not fixed and is reproducible in the latest CI (**[macOS](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_macos.yaml?query=branch%3Amaster+event%3Apush)** / **[Ubuntu](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_ubuntu.yaml?query=branch%3Amaster+event%3Apush)** / **[Windows](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_windows.yaml?query=branch%3Amaster+event%3Apush)**) "master" builds.

Reply:
> 
> What about this? (marked as not needing validation, of course!)
> 
> > * [ ]   (optional but recommended) I have verified that the issue is not fixed and is reproducible in the latest CI (**[macOS](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_macos.yaml?query=branch%3Amaster+event%3Apush)** / **[Ubuntu](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_ubuntu.yaml?query=branch%3Amaster+event%3Apush)** / **[Windows](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci_windows.yaml?query=branch%3Amaster+event%3Apush)**) "master" builds.

It's also good 👍  

Let's see what the others think.

Reply:
I will also change the title from "Mandatory prerequisites" to just "Prerequisites" if we are going to go back & include the (optional but recommended) part.

Reply:
> 
> 
> I will also change the title from "Mandatory prerequisites" to just "Prerequisites" if we are going to go back & include the (optional but recommended) part.

Agreed 👍

Reply:
>What about one of these?

IMO, "master" has no meaning for regular users.

Reply:
> IMO, "master" has no meaning for regular users.

Will omit.

Reply:
addressed.